Remotion/ObjectBinding/Web/UI/Controls/BocTreeView.cs

```diff
@@ -518,14 +534,21 @@ protected virtual string GetToolTip (IBusinessObjectWithIdentity businessObject)
               && referenceProperty.IsList
               && referenceProperty.ReferenceClass is IBusinessObjectClassWithIdentity
               && referenceProperty.IsAccessible (parentBusinessObject))
-            referenceListPropertyInfos.Add (new BusinessObjectPropertyTreeNodeInfo (referenceProperty));
+            referenceListPropertyInfos.Add (CreateBusinessObjectPropertyTreeNodeInfo (referenceProperty));
         }
         return (BusinessObjectPropertyTreeNodeInfo[]) referenceListPropertyInfos.ToArray (typeof (BusinessObjectPropertyTreeNodeInfo));
       }
 
-      return new[] { new BusinessObjectPropertyTreeNodeInfo (Property) };
+      return new[] { CreateBusinessObjectPropertyTreeNodeInfo (Property) };
     }
 
+    private BusinessObjectPropertyTreeNodeInfo CreateBusinessObjectPropertyTreeNodeInfo (IBusinessObjectReferenceProperty property)
+    {
+      var businessObjectPropertyTreeNodeInfo = new BusinessObjectPropertyTreeNodeInfo (property);
+      businessObjectPropertyTreeNodeInfo.Badge = GetBadge (Property);
```

Comment:
Integrating `GetBadge` when creating the Property node was still missing (see old comments)